### DISCUSSION TITLE
Simple Static Error

### DISCUSSION THREAD

**Discussion by memonkamran5522** [community/community#136439] [Programming Help]
### Select Topic Area

Question

### Body

I'm using the Simply Static plugin to generate a static version of my site, but I'm encountering an error during the process. The error message reads:

"[2024-08-21 06:49:31] Error: (1) Maximum execution time of 120 seconds exceeded in C:\xampp\htdocs\Yoursitename\wp-includes\class-wpdb.php on line 2344."

If anyone has a solution for this issue, I'd really appreciate your help. Thanks!

**Comment by arhis222** [comment]


**Reply by AbuBakkarRaihan** [reply]
copy paste 😂 chatgpt!

**Reply by arhis222** [reply]
Locate .htaccess and then add the following line at the end of the file:
php_value max_execution_time 300

**Reply by arhis222** [reply]
nope I had the same issue and I followed this step

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @memonkamran5522 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
